devnets/startNetwork.ts

```diff
@@ -361,3 +346,8 @@ function addXcmHrmpChannels(
     }
   }
 }
+
+async function getLibp2pPeerId(privateKey: Uint8Array) {
+  const publicKey = await ed25519.getPublicKeyAsync(privateKey)
+  return base58.encode(new Uint8Array([0, 36, 8, 1, 18, 32, ...publicKey]))
```

Comment:
Can you add a comment explaining the derivation of these numbers / linking to an explanation?